docs(gateway): cleanup references to old services (#1116)
kumahq/kuma-website: app/docs/dev/explore/gateway.md

### Usage

> Let's create and configure a gateway that listens for traffic from outside of our mesh
> and forwards it to the [demo app frontend](https://kuma.io/docs/dev/quickstart/kubernetes/).

Reviewer: ```suggestion
You can create and configure a gateway that listens for traffic from outside of your mesh
and forwards it to the [demo app frontend](https://kuma.io/docs/dev/quickstart/kubernetes/).
```

See the [dedicated section](/docs/{{ page.version }}/deployments/multi-zone) for detailed information.
{% endtip %}

> Now let's define a `MeshGatewayRoute` to forward our traffic based on the
> matched URL path.

Reviewer: ```suggestion
Now, you can define a `MeshGatewayRoute` to forward your traffic based on the
matched URL path.
```
